account_invoice_mass_sending/models/account_invoice.py

```diff
@@ -0,0 +1,76 @@
+# Copyright 2019 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import _, api, fields, models
+from odoo.addons.queue_job.job import job
+from odoo.exceptions import UserError
+
+JOB_QUEUE_CHANNEL = "root.PREPARE_SEND_PRINT_INVOICE"
```

Comment:
Hi @RobinetDenisAcsone ,

In V10, only the "send by email" feature is assumed by the final wizard method invoked (i.e. `send_mail_action()`).

Thus, IMO, everywhere the term "Print" is used as a part of a name or a label (intended to the final user), il should be deleted.